src/extension/interop/ExtensionInterop.h

```diff
@@ -22,6 +22,8 @@ extern "C"
     /// \return The version number
     [[maybe_unused]] API_EXPORT int32_t ExtensionGetVersion(ThriveConfig* thriveConfig);
 
+    [[maybe_unused]] API_EXPORT bool Unwrap(float texelSize, ArrayMesh* mesh);
```

Comment:
For consistency I'd prefer to swap the arguments here so that the object the method is called on is always first. Also as this is a C file the function name "Unwrap" is the *entire* thing that resolves the method (not the parameter names) so having this short of a function name is not great here. I'd suggest naming this `ArrayMeshUnwrap(ArrayMesh* mesh, float texelSize)`